src/scripts/train/OLMo2-1B.py

```diff
@@ -73,6 +76,48 @@ def build_trainer_config(common: CommonComponents) -> TrainerConfig:
                 cancel_check_interval=10,
             ),
         )
+        .with_callback(
+            "downstream",
```

Comment:
Let's call this "downstream_evaluator" to be consistent with the naming convention for other callbacks.
```suggestion
            "downstream_evaluator",
```

Reply:
thanks, fixed